library/src/common/utils.ts

```diff
@@ -294,6 +300,125 @@ const getQueryParams = <Q>(query: Q): string => {
   return '';
 };
 
+const appInfoToUserAgentString = (appInfo: ApplicationInfo): string =>
+  `(${appInfo.name || ''}; ${appInfo.version || ''}; ${appInfo.url || ''})`;
+
+const buildUserAgentString = (appInfo?: ApplicationInfo): string => {
+  let userAgent = `${USER_AGENT_CLIENT}/${process.env.VERSION || 'unknown'}`;
+
+  if (appInfo && Object.keys(appInfo || {}).length) {
+    userAgent += ` ${appInfoToUserAgentString(appInfo)}`;
+  }
+
+  return userAgent;
+};
+
+const getBrowser = (): string => {
+  const { userAgent } = window.navigator;
+
+  let version = 'unknown';
+
+  const browser = [
```

Comment:
This const could be externalized. I don't know why the linter is not complaining about it.